collect_translations.sh

```diff
@@ -7,14 +7,24 @@ if [[ ! -z $1 ]] ; then
 fi
 
 # .tsl = translation string list
-echo "COLLECTING TRANSLATIONS"
+echo "-== COLLECTING TRANSLATIONS ==-"
 # gather first argument of TRANSLATE_ME
-echo "" >"${output}.tsl"
-for files in $(grep -rsIl --include="*.c" --include="*.cc" --include="*.cpp" --include="*.ecpp" --include="*.h" --include="*.hpp" --include="*.inc" --exclude-dir=".build" --exclude-dir=".srcclone" --exclude-dir=".install" TRANSLATE_ME "${target}"); do tr -d '\n' <"${files}" | sed 's/#define *TRANSLATE_ME//g;s/TRANSLATE_ME *(/\n/g' | sed 's/\([^\]\)".*$/\1/' | sort | uniq >"${output}.tsl" ; done
+echo "" >"${output}.ttsl"
+for files in $(grep -rsIl --include="*.c" --include="*.cc" --include="*.cpp" --include="*.ecpp" --include="*.h" --include="*.hpp" --include="*.inc" --exclude-dir=".build" --exclude-dir=".srcclone" --exclude-dir=".install" TRANSLATE_ME "${target}"); do
```

Comment:
`for FILE` ? each instance of the variable points to one filename (not file*s*), and it is customary to CAPS the varnames to differentiate easier from commands and keywords